src/Piece.js

```diff
@@ -1,5 +1,35 @@
 /* Represents a piece in the game */
 class Piece 
```

Comment:
Should define a Create() method similar to Vector. Should accept a JSON object based on a notation decided for the config files. This will also either include parsing the ```m``` flag or insist on three lists of vectors. Assume player comes in as a string, for sanity.